scripts/exregional_process_bufr.sh

```diff
@@ -0,0 +1,386 @@
+  print_err_msg_exit "\
+The executable specified in exec_fp does not exist:
+  exec_fp = \"${exec_fp}\"
+Build lightning process and rerun."
+fi
+#
+#
+#-----------------------------------------------------------------------
+#
+# Run the process for lightning bufr file 
+#
+#-----------------------------------------------------------------------
+#
+if [[ "$run_lightning" == true ]]; then
+   PREP_STEP
+   eval $RUN_CMD_UTILS ${exec_fp} ${REDIRECT_OUT_ERR} || print_err_msg "\
+        Call to executable to run lightning process returned with nonzero exit code."
+   POST_STEP
+fi
+
+#
+#-----------------------------------------------------------------------
+#-----------------------------------------------------------------------
+#
+# Link to the observation NASA LaRC cloud bufr file
+#
+#-----------------------------------------------------------------------
+
+obs_file=${obspath_tmp}/${obsfileprefix}.t${HH}${SUBH}z.lgycld.tm00.bufr_d
+print_info_msg "$VERBOSE" "obsfile is $obs_file"
+run_cloud=false
+if [ -r "${obs_file}" ]; then
+   cp_vrfy "${obs_file}" "lgycld.bufr_d"
+   run_cloud=true
+else
+   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
+fi
+
+#-----------------------------------------------------------------------
+#
+# Build namelist and run executable for NASA LaRC cloud
+#
+#   analysis_time : process obs used for this analysis date (YYYYMMDDHH)
+#   bufrfile      : result BUFR file name
+#   npts_rad      : number of grid point to build search box (integer)
+#   ioption       : interpolation options
+#                   = 1 is nearest neighrhood
+#                   = 2 is median of cloudy fov
+#   bkversion     : grid type (background will be used in the analysis)
+#                   = 0 for ARW  (default)
+#                   = 1 for FV3LAM
+#-----------------------------------------------------------------------
+
+if [ ${PREDEF_GRID_NAME} == "GSD_RAP13km" ]; then
+   npts_rad_number=1
+   metar_impact_radius_number=9
+else
+   npts_rad_number=3
+   metar_impact_radius_number=15
+fi
+
+cat << EOF > namelist.nasalarc
+ &setup
+  analysis_time = ${YYYYMMDDHH},
+  bufrfile='NASALaRCCloudInGSI_bufr.bufr',
+  npts_rad=$npts_rad_number,
+  ioption = 2,
+  grid_type = "${PREDEF_GRID_NAME}",
+ /
+EOF
+
+#
+#-----------------------------------------------------------------------
+#
+# Copy the executable to the run directory.
+#
+#-----------------------------------------------------------------------
+#
+exec_fn="process_larccld.exe"
+exec_fp="$EXECdir/${exec_fn}"
+
+if [ ! -f "${exec_fp}" ]; then
+  print_err_msg_exit "\
+The executable specified in exec_fp does not exist:
+  exec_fp = \"${exec_fp}\"
+Build lightning process and rerun."
+fi
+#
+#
+#-----------------------------------------------------------------------
+#
+# Run the process for NASA LaRc cloud  bufr file 
+#
+#-----------------------------------------------------------------------
+#
+if [[ "$run_cloud" == true ]]; then
```

Comment:
If this line of the if statement was moved to line 243, it will be more efficient since the executable isn't meant to run.